android/app/build.gradle

```diff
@@ -145,6 +145,7 @@ dependencies {
     compile fileTree(dir: "libs", include: ["*.jar"])
     compile "com.android.support:appcompat-v7:23.0.1"
     compile "com.facebook.react:react-native:+"  // From node_modules
+    compile "com.facebook.fresco:animated-gif:1.3.0"
```

Comment:
Just a suggestion: you can add comment as similar to 147 line for clarity:
```
// For animated GIF support
```

Reply:
@lex111 I thought so, but I find the library name quiet explicit.